src/OpenTK/Properties/AssemblyInfo.cs

```diff
@@ -9,6 +10,9 @@
 [assembly: AssemblyFileVersionAttribute("3.0.0")]
 [assembly: CLSCompliantAttribute(true)]
 [assembly: AssemblyCopyrightAttribute("Copyright (c) 2006 - 2016 Stefanos Apostolopoulos <stapostol@gmail.com> for the Open Toolkit library.")]
+
+[assembly: InternalsVisibleTo("OpenTK.GLWidget, PublicKey=0024000004800000940000000602000000240000525341310004000011000000a3e05aafb87f71fb8fd02b512707f289c12341de98c6ce2ec1494b71c20cb2032cbd65d091b447df3ec772257efb9fa3591201937890e067da1d29a339948a12b29c2847a787cc9fbef2a3bf78267026e85f36aab827228df4bb580e3ae0599ade036f0a97a0e6982f5406d98d114455f332350f6d8369db655e9c3e7976c2c8")]
```

Comment:
I recall there being a discussion of modularity where the project will have a "core" and the additional features could be included as needed.

If that's still a goal, this strong coupling may be a cause for concern.

Reply:
Right. In this particular case, the widget project is still completely modular, but will not function without the main OpenTK library. The main OpenTK library does not depend on the GLWidget project by allowing it to access internals - it's just a bit of metadata. 

That being said, with some additional changes I've not yet pushed, marking it with InternalsVisibleTo is no longer required, and will be removed.